tools/wheels/cibw_before_build_linux.sh

```diff
@@ -13,13 +13,13 @@ else
     exit 1
 fi
 
-PLATFORM=$(PYTHONPATH=tools python -c "import openblas_support; print(openblas_support.get_plat())")
-
 printenv
 # Update license
 cat $PROJECT_DIR/tools/wheels/LICENSE_linux.txt >> $PROJECT_DIR/LICENSE.txt
 
 # Install Openblas
-basedir=$(python tools/openblas_support.py $NIGHTLY_FLAG)
-cp -r $basedir/lib/* /usr/local/lib
-cp $basedir/include/* /usr/local/include
+echo PKG_CONFIG_PATH $PKG_CONFIG_PATH
+PKG_CONFIG_PATH=$PROJECT_DIR/.openblas
+rm -rf $PKG_CONFIG_PATH
+mkdir -p $PKG_CONFIG_PATH
```

Comment:
The `rm -rf` and `mkdir` at least should no longer be needed. Also maybe change `PKG_CONFIG_PATH` to simply `$PWD`, so the separate `.openblas` directory isn't needed? A single `.pc` file can live anywhere, no need for a subdir.

Reply:
adopted